src/app/f2f/steps.js

```diff
@@ -83,12 +83,12 @@ module.exports = {
         field: "ukPassportExpiryDate",
         op: "before",
         value: "18 months ago",
-        next: "photoIdExpiry",
+        next: "photo-id-expired",
       },
-      "findBranch",
+      "find-post-office-prove-identity",
     ],
   },
-  "/nonUKPassportDetails": {
+  "/non-uk-passport-expire": {
```

Comment:
Can I ask why we're not using the PATH constants from the config files for these? Would mean we only have to change in one place in the future

Reply:
The step paths need to be strings but I've updated the k/v pairs inside the objects to use the path constants

Reply:
I might be pushing my luck here so tell me if you disagree, but something like this _should_ work and would allow us to use the config for the paths?

```
  [`/${APP.PATHS.NON_UK_PASSPORT_DETAILS]}`: {
    fields: ["nonUKPassportExpiryDate"],
    controller: nonUKPassportDetails,
    editable: true,
    editBackStep: APP.PATHS.CHECK_DETAILS,
    next: [
      {
        field: "nonUKPassportExpiryDate",
        op: "before",
        value: "today",
        next: APP.PATHS.EXPIRED_ID,
      },
      APP.PATHS.NON_UK_PASSPORT_COUNTRY_SELECTOR,
    ],
  },
```

Reply:
No you're absolutely right thank you, I've changed them all and repushed!

Reply:
Sorry I thought we had to append a `/` but it doesn't look like we do! In that case just `[APP.PATHS.NON_UK_PASSPORT_DETAILS]` should be enough